src/gg.regression.unity.bots/Runtime/Scripts/RGOverlay/RGSequenceManager.cs

```diff
@@ -135,6 +147,35 @@ public void DeleteSequenceByPath(string path)
         LoadSequences();
     }
 
+    /**
+     * <summary>
+     * When this component is viewed on a mobile device:
+     * - Scale the entire canvas so that all components are larger
+     * - Disable the reload Sequence/Segment button
+     * - Hide the create Sequence button
+     * </summary>
+     */
+    public void SetMobileView()
+    {
+        GetComponent<CanvasScaler>().referenceResolution = new Vector2(800, 600);
```

Comment:
From what I'm reading, since our Canvas Scaler component has its `match` field set to `0.5`, the scaling will be the same if the viewport is 800 x 600 or 600 x 800. In practice it seems to scale the same
![image](https://github.com/user-attachments/assets/ab1383a5-6f84-4816-8552-ef5ff2a96192)

Reply:
For clarity regarding clarity on portrait vs mobile concerns, and since the first operation of this function is identical to that of SetPortraitView, I wonder if it may be better to instead do the following:

1. Rename `SetPortraitView()` to "UpscaleUI" or something just to represent the operation of making things better.
2. Then in this `SetMobileView()` function, you deactivate things

Not something I'm too worried about so this is a nit, but it could make things more clear wrt to the comments that Zack had

Reply:
@vontell good call on the function rename, done 👍🏻 

@RG-nAmKcAz the portrait and mobile connection is that I'm making it so that the UI is scaled to be larger in both scenarios. When running a build on mobile, scale the UI up (and change a few other small things). When running a game in Unity + the viewport is in a portrait orientation, scale the UI up.

The overall goal is just to make the UI a bit larger in situations where it becomes harder to use (eg: on a small screen). Hopefully I've made some sense